recipe/patches/0001_remove_safelocale.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
```

Comment:
header on the patch to explain would be good

Reply:
Done!